ecs-agent/api/ecs/client/ecs_client.go

```diff
@@ -506,6 +506,11 @@ func (client *ecsClient) SubmitTaskStateChange(change ecs.TaskStateChange) error
 }
 
 func (client *ecsClient) submitTaskStateChange(change ecs.TaskStateChange) error {
+
+	clusterARN := client.configAccessor.Cluster()
+	if len(change.ClusterARN) != 0 {
```

Comment:
nit: Prefer to use `change.ClusterARN != ""` in favor of `len(change.ClusterARN) != 0` for readability.

Reply:
Sure. Will change in a follow-up CR.